projects/src/simbicycle.hh

```diff
@@ -161,6 +162,21 @@ void Bicycle<T, U, V>::update_kinematics() {
     m_pose.steer = steer;
 }
 
+template <typename T, typename U, typename V>
```

Comment:
If T, U and V are of a specific .. category, for lack of a better word, then maybe it makes sense to name them less abstractly?